.all-contributorsrc

```diff
@@ -41,6 +41,20 @@
         "video"
       ]
     },
+    {
```

Comment:
minor comments about the contributions file:
* please move your record to the end. The entries are roughly chronological.
* `code` is fine, thanks for your contribution!
* `maintenance` is fine since this concerns the CI/release
* `doc` cannot be added with PR. You can add it with the other PR related to docstrings.
* `test` is not part of this PR.
* `ideas` is not part of this PR
* `question` is not justified afaik since you have not answered user or contributor questions on the community channels (yet)